src/core/client/admin/routes/Configure/sections/Auth/LocalAuthConfigContainer.tsx

```diff
@@ -0,0 +1,99 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+import { graphql } from "react-relay";
+
+import { withFragmentContainer } from "coral-framework/lib/relay";
+import { GQLFEATURE_FLAG } from "coral-framework/schema";
+import { Icon } from "coral-ui/components/v2";
+import { CallOut } from "coral-ui/components/v3";
+
+import Header from "../../Header";
+import ConfigBoxWithToggleField from "./ConfigBoxWithToggleField";
+import RegistrationField from "./RegistrationField";
+import TargetFilterField from "./TargetFilterField";
+
+import styles from "./LocalAuthConfig.css";
+
+// eslint-disable-next-line no-unused-expressions
+graphql`
+  fragment LocalAuthConfigContainer_formValues on Auth {
+    integrations {
+      local {
+        enabled
+        allowRegistration
+        targetFilter {
+          admin
+          stream
+        }
+      }
+    }
+  }
+`;
+
+interface Props {
+  disabled?: boolean;
+  settings: any;
```

Comment:
Change this to `settings: LocalAuthConfigContainer_settings`, import the types and we can merge this! :-)